packages/core/src/Stores/client-store.js

```diff
@@ -1644,11 +1645,12 @@ export default class ClientStore extends BaseStore {
                 BinarySocketGeneral.authorizeAccount(authorize_response);
 
                 // Client comes back from oauth and logs in
-                RudderStack.identifyEvent(user_id, {
+                Analytics.identifyEvent(user_id, {
                     language: getLanguage().toLowerCase(),
+                    app_id: getAppId(),
                 });
                 const current_page = window.location.hostname + window.location.pathname;
-                RudderStack.pageView(current_page);
+                Analytics.pageView(current_page);
```

Comment:
Perhaps we need to add this function in Analytics if we really need use it
let me do it in my pr for package